cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -425,7 +443,7 @@ module {{bicepName .Name}} 'br/public:avm/res/app/container-app:0.8.0' = {
           {{- end}}
           {{- if (and .AzureEventHubs .AzureEventHubs.AuthUsingManagedIdentity) }}
           {
-            name: 'SPRING_CLOUD_AZURE_EVENTHUBS_CONNECTION_STRING'
+            name: 'SPRING_CLOUD_AZURE_EVENTHUBS_CONNECTIONSTRING'
```

Comment:
FYI: Both `SPRING_CLOUD_AZURE_EVENTHUBS_CONNECTION_STRING` and `SPRING_CLOUD_AZURE_EVENTHUBS_CONNECTION_STRING` can work, but `SPRING_CLOUD_AZURE_EVENTHUBS_CONNECTION_STRING` is the legacy format.

Reply:
Did you mean `SPRING_CLOUD_AZURE_EVENTHUBS_CONNECTIONSTRING`?

Reply:
Yes. you are right. I updated the comment.